MyFirstList.java

```diff
@@ -11,11 +11,13 @@ public static void main(String[] args) {
 		NUM NUM1 = new NUM (33);

```

Comment:
For example, class name should be in CamelCase 
https://en.wikipedia.org/wiki/Camel_case
aka Num